File: vane-regions/src/main/java/org/oddlama/vane/regions/menu/RegionGroupMenuTag.java
package org.oddlama.vane.regions.menu;

import java.util.UUID;

public class RegionGroupMenuTag {

    private final UUID regionGroupId;

    public RegionGroupMenuTag(final UUID regionGroupId) {
        this.regionGroupId = regionGroupId;
    }

    public UUID regionGroupId() {
        return regionGroupId;
    }
}

File: vane-regions/src/main/kotlin/org/oddlama/vane/regions/menu/RegionGroupMenuTag.kt
package org.oddlama.vane.regions.menu

import java.util.*

/**
 * Lightweight menu tag carrying a selected region-group id.
 */
class RegionGroupMenuTag(private val regionGroupId: UUID?) {
    /**
     * Returns the associated region-group id.
     */
    fun regionGroupId(): UUID? = regionGroupId
}
